CHANGELOG.rst

```diff
@@ -1,5 +1,40 @@
 .. _changelog:
 
+0.43.1
```

Comment:
0.43.0 has not been tagged yet...

Reply:
Ah, right... I added the changes to 0.43.0